pkg/graph/resolvers/resolver_test.go

```diff
@@ -50,15 +54,57 @@ func (suite *ResolverSuite) createDiscussionReply(pd *models.PlanDiscussion, con
 	return dr
 }
 
-func (suite *ResolverSuite) createPlanCollaborator(mp *models.ModelPlan, EUAUserID string, fullName string, teamRole models.TeamRole, email string) *models.PlanCollaborator {
+func (suite *ResolverSuite) createPlanCollaborator(mp *models.ModelPlan, EUAUserID string, fullName string, teamRole models.TeamRole, emailAddress string) *models.PlanCollaborator {
 	collaboratorInput := &model.PlanCollaboratorCreateInput{
 		ModelPlanID: mp.ID,
 		EuaUserID:   EUAUserID,
 		FullName:    fullName,
 		TeamRole:    teamRole,
-		Email:       email,
+		Email:       emailAddress,
+	}
+
```

Comment:
`Non-blocking`  I'd argue that doing a unit test here is perhaps redundant with the `plan_collaborator_test.go` file. I think this function is just meant to be a utility to test other resolver code, and we test the collaborator in the other file.